awx/ui_next/src/screens/Inventory/InventorySources/InventorySourceList.test.jsx

```diff
@@ -215,6 +215,47 @@ describe('<InventorySourceList />', () => {
 
     expect(wrapper.find('ContentError').length).toBe(1);
   });
+  test('should render sync all button and make api call to start sync for all', async () => {
+    const readSourcesResponse = {
+      data: {
+        results: [
+          {
+            id: 1,
+            name: 'Source Foo',
+            status: '',
+            source: 'ec2',
+            url: '/api/v2/inventory_sources/56/',
+            summary_fields: {
+              user_capabilities: {
+                edit: true,
+                delete: true,
+                start: true,
+                schedule: true,
+              },
+            },
+          },
+        ],
+        count: 1,
+      },
+    };
+    InventoriesAPI.readSources
+      .mockResolvedValue({ ...readSourcesResponse, status: 'pending' })
+      .mockResolvedValueOnce(readSourcesResponse);
+    InventorySourcesAPI.readOptions.mockResolvedValue({
```

Comment:
Do we need to mock `readOptions` and `readSources` again in this test?

Reply:
No, I went down the road of trying to ensure that the sync buttons for the list item change properly in this test but decided that I had covered that functionality sufficiently in the InventorySourceLIstItem test file and failed to remove these mocks.  Good catch